src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy

```diff
@@ -3195,14 +3211,22 @@ public enum LegendsAwakened implements LogicCardInfo {
           pokePower "Trade Off", {
```

Comment:
This power isn't fully implemented. You can't use more than 1 Trade Off per turn, so you have to make sure other Pokémon with the same power can't use it. Check the Aegislash in RebelClash for an example on bwAbility.

Reply:
Pretty sure I fixed this correctly now

Reply:
Yeah, I think it looks good now.